pages/pads.md

```diff
@@ -11,9 +11,9 @@ to hillhackers on the venue. We have the following pads listed:
 ### 2017
 
 {% for pad in site.data.pads['2017'] %}
-- [{{pad[0]}}](https://pads..in/{{pad[1]}})
```

Comment:
To me this looks like it wanted to be https://pads.hackbeach.in/{{pad[1]}}.

Reply:
Makes more sense. :)